### DISCUSSION TITLE
Testing BPX61

### DISCUSSION THREAD

**Discussion by florenceol** [ozel/DIY_particle_detector#33] [Q&A]
I think I may have damaged the BPX61 when I removed the glass.  Can you please recommend a good way to test the BPX61 with a multimeter.  Thanks.

**Comment by ozel** [comment]
that is very easy: just connect a multimeter across the pins and hold the diode against light, then play with your hand in front of the diode making shadow.
If you do not see some 100 Millivolt change with light versus shadow it is damaged.
This works even if the diode is already soldered onto the board. 
Have you followed the troubleshooting steps? Light is in general good for testing the whole detector:
https://github.com/ozel/DIY_particle_detector/wiki/Troubleshooting

If you measure only 0V even with a lot of light and the diode is already soldered on the board, you should check if there is a short circuit between both diode pins.
This actually may happen if the diode is completely resting on the board, fully touching it. 
The whole BPX61 diode case (the part that supports the glass window) should have a small distance, like 1 mm, to the board to prevent shorting of the pins. Just heat both diode pins with your solder iron again and lift it up by a small amount, away from the board surface and try again to measure light. You can ask a second person to pull the diode with pliers while you are heating the pins with the soldering iron. Mounting the board in a fixture or vice is even better for that procedure.

**Reply by florenceol** [reply]
Thanks for your fast reply.  The BPX61 tested OK.  I followed the troubleshooting.  I got 10K ohms across the battery connector.  I verified the polarity of the electrolytic cap, the photodiode and the opamp.  Does it matter which pads you use for the photodiode (D1, D2, D3 or D4)?  One photo shows the BPX61 on [D3](http://physicsopenlab.org/wp-content/uploads/2020/06/CERNDet-768x955.jpg).  Another photo looks like either [D1 or D4](https://github.com/ozel/DIY_particle_detector/raw/master/hardware/V1.2/documentation/3D_top_alpha.png).

**Reply by ozel** [reply]
no, the choice of diode pads does not matter. what happens if you switch on/off? 
you should see a spike in the output.

The detector also must react to light (but not too much of it, it quickly saturates).
Also please use a real oscilloscope for troubleshooting, not a soundcard.

Please post pictures of the board, the metal case that you chose and your whole test setup. 
What test source are you using?
Without all this info, it is impossible to give guidance. You can just drag and drop pictures into the text field here.
